io.github.KangLin.RabbitRemoteControl.json

```diff
@@ -0,0 +1,241 @@
+{
+    "id": "io.github.KangLin.RabbitRemoteControl",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.8",
+    "sdk": "org.kde.Sdk",
+    "command": "RabbitRemoteControlApp",
+    "cleanup": [
+        "*.la",
+        "*.a",
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig"
+    ],
+    "finish-args": [
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=pcsc",
+        "--socket=ssh-auth",
+        "--socket=x11",
+        "--device=all",
+        "--filesystem=xdg-documents",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.secrets",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--system-talk-name=org.freedesktop.Avahi"
+    ],
+    "cleanup-commands": [
+        "rm -fr /var/tmp/*"
```

Comment:
just add this to cleanup?

Reply:
the root cause should be addressed instead. There is no reason it copies anything in /var/tmp during build.